test/utilities/update_dom_by_api_request_test.js

```diff
@@ -0,0 +1,86 @@
+require("../setup");
+const utilities = require('../../app/javascript/src/utilities.js');
+
+
+describe('utilities.updateDomByApiRequest', function() {
+  var targetId = "update-dom-by-api-request-target-element";
+  var insertId = "update-dom-by-api-request-insert-element";
+  var get;
+
+  before(function() {
+    get = $.get;
+    $(document.body).append("<p id=\"" + targetId + "\"></p>");
+    $.get = function(urlNotNeeded, response) {
+      response("<span id=\"" + insertId + "\">Luke</span>");
+    }
+  });
+
+  afterEach(function() {
+    $(document.body).find("#" + insertId).remove();
+  });
+
+  after(function() {
+    $(document.body).find("#" + targetId).remove();
+    $.get = get;
+  });
+
+  it('should place new element after target node', function() {
+    var $targetNode = $(document.body).find("#" + targetId);
+
+    // First check inserted node is not there.
+    expect($(document.body).find("#" + insertId).length > 0).is.false;
+
+    utilities.updateDomByApiRequest("", {
+      target: $targetNode,
+      done: function() {
+        var $insertedNode = $targetNode.find("#" + insertId);
+        expect($targetNode).to.exist;
+        expect($targetNode.length).to.equal(1);
+        expect($insertedNode).to.exist;
+        expect($insertedNode.length).to.equal(1);
+        expect($targetNode.find($insertedNode).length > 0).to.be.true;
+      }
+    });
+  });
+
+  it('should place new element before target node', function() {
```

Comment:
Think this should be `it('should place new element after target node)` as you have type `after` in the `updateDomByApiRequest` call, and you're expectation is `($targettnode).next()`

Reply:
Correct. Have fixed typo.